index.html

```diff
@@ -14,20 +16,52 @@
     <div class="container">
         <header class="header-page">
             <!--header contents goes here-->
-            <nav>
+            <nav class="navbar">
                 <!--This is my Logo-->
-                <a class="logo" href=''>Solorics</a>
+                <a class="logo" href='#'>Solorics</a>
                 <ul class="nav-links">
-                    <li><a href="#portfolio">Portfolio</a></li>
-                    <li><a href="#contacts">Contaacts</a></li>
-                    <li><a href="about">About</a></li>
-                    <li><a href="#mailbox"><img src="./Images/Union.png" alt="mail_box"></a></li>
+                    <li class="nav-item"><a class="menuItem" href="#portfolio">Portfolio</a></li>
+                    <li class="nav-item"><a class="menuItem" href="#contact">Contacts</a></li>
+                    <li class="nav-item"><a class="menuItem" href="#about">About</a></li>
+                    <li class="nav-item"><a class="menuItem" href="#mailbox"><img src="./Images/Union.png" alt="mail_box"></a></li>
                 </ul>
-                <!--Toggle button goes here-->
-                <img src="./Images/burger.png" alt="Toggle_Image_Button" class="toggle">
+                <!-- <div id="myNav" class="hamburger" onclick="openNav()">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div> -->
+                <nav>
+                    <img class="toggle" src="./Images/toggle.png" alt="toggle" onclick="openNav()">
```

Comment:
### Optional
- [ ] It would be better if we add a little user experience to our hamburger button, like for example, when a user hovers over the button the mouse should show a pointer so that it can indicate on press activity